package.json

```diff
@@ -28,8 +28,10 @@
     "jsonwebtoken": "^8.3.0",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
+    "mongoose": "^5.4.15",
+    "mongoose-unique-validator": "^2.0.1",
```

Comment:
Why do we still have mongoose, I thought we are making use of PostgreSQL and sequelize, could you kindly confirm and make the necessary changes

Reply:
I've uninstalled mongoose now.